### DISCUSSION TITLE
Public Link not working on Kubernetes Cluster

### DISCUSSION THREAD

**Discussion by cunyeune** [outline/outline#6061] [Self Hosting]
I am currently deploying an outline on a Kubernetes cluster. Everything was working fine until today when I checked the public links shared outside for non-logged-in users, they all reported 'Not Found' errors. I'm not sure what the cause is, please help.

![image](https://github.com/outline/outline/assets/121747048/35248641-3abb-452f-affc-62199a697da4)
![image](https://github.com/outline/outline/assets/121747048/0438ad3a-f1cd-41f1-ad3f-5a928934a646)

**Comment by tommoor** [comment]
Are you sure public sharing wasn't disabled?

**Reply by cunyeune** [reply]
Yes, I'm sure I've enabled the public sharing feature. I just added images for description, I apologize for the oversight.
